.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+	extends: [
+		'wikimedia',
+		'wikimedia/node',
+	],
+	rules: {
+		'comma-dangle': [ 'error', {
```

Comment:
Minor (feel free to ignore): `error` level shows as a bright red color in my IDE. I think errors should be reserved for probable bugs so that style concerns are only tested on commit. During development, it should be A-OK to write sloppy code in my view. As far as I know, [comma-dangle](https://eslint.org/docs/rules/comma-dangle) is a runtime concern for the unsupported IE8 browser and a style concern for everyone else. For this reason, I recommend lowering this to `'warn'` and revising the NPM script from `eslint .` to `eslint --cache --max-warnings 0 .`.

Reply:
That's my understanding. If the cache doesn't invalidate properly, it would be a broken cache. We've been using it in MobileFrontend for some time without issue.